.github/workflows/pullrequest_check.yml

```diff
@@ -59,15 +61,14 @@ jobs:
         run: yarn run check
 
   integration_and_unit_tests:
-    if: github.event.pull_request.draft == false
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v2
       - name: libudev-dev
         run: |
           sudo apt-get update
-          sudo apt-get install -y libudev-dev
+          sudo apt-get install -y libudev-dev python3
```

Comment:
I think we can move installing python to within running the cli tests job, because few PRs only will have the build CLI label. And the other PRs shouldn't have to install python each run (Although it may be installed already on ubuntu)

Reply:
Agreed. I have made the changes.